src/framework/theme/components/tabset/tab-label.ts

```diff
@@ -0,0 +1,7 @@
+import { Directive } from '@angular/core';
+
+/** Used to flag tab labels */
+@Directive({
+  selector: '[nbTabLabel]',
+})
+export class NbTabLabelDirective {}
```

Comment:
Let's change label to title as we reference it in tab object.
```suggestion
export class NbTabTitleDirective {}
```